ikasaneip/sample/spring-boot/jms/src/test/java/com/ikasan/sample/spring/boot/builderpattern/JmsSampleFlowTest.java

```diff
@@ -311,18 +344,34 @@ public void test_exclusion_followed_by_resubmission()
         this.flowTestRule.startFlow();
 
         boolean result = resubmissionRestService.resubmit("http://localhost:" + this.randomServerPort + this.moduleUnderTest.getName(), this.moduleUnderTest.getName(),
-            "Jms Sample Flow", "resubmit",  exclusionEvent.getErrorUri());
+            "Jms Sample Flow", "resubmit", exclusionEvent.getErrorUri());
 
         assertEquals(true, result);
 
         exclusions = exclusionManagementService.find(null, null, null, null, null, 100);
         assertEquals(0, exclusions.size());
     }
 
-    @DirtiesContext
+    private void assertErrorsWithWait(int expectedNumberOfErrors) {
```

Comment:
Can this be Up streamed to IkasanFlowTestRule

to be able to call flowTestRule.assertErrorsWithWait(maxWaitTime,ExpectedErrorNumber)

Reply:
see previous